baseapp/baseapp_test.go

```diff
@@ -97,6 +98,14 @@ func registerTestCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&msgNoRoute{}, "cosmos-sdk/baseapp/msgNoRoute", nil)
 }
 
+// txEncoder creates a amino TxEncoder for testing purposes.
+func txEncoder() sdk.TxEncoder {
```

Comment:
We should have the name be a bit more descriptive of its use here. Maybe `aminoTxDecoder`?